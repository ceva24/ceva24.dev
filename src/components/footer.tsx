import React from "react";
import { Link } from "gatsby";
import { StaticImage } from "gatsby-plugin-image";

const Footer: React.FC = () => (
    <footer>
        <div className="min-w-max text-center">
            <Link to="/">Home</Link>
        </div>

        <div className="my-8 text-center space-y-3 pt-5 border-t max-w-lg mx-auto">
            <StaticImage
                src="../img/profile-picture.png"
                alt="Chris Evans profile picture"
                quality={100}
                className="rounded-full"
            />

            <div className="font-normal">CHRIS EVANS</div>

            <div className="px-6">
                A Web Development / Systems Integration Team Leader at the
                University of York, UK
            </div>
        </div>
    </footer>
);

export { Footer };
