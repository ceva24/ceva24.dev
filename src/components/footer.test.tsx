import React from "react";
import { render } from "@testing-library/react";
import { Footer } from "./footer";

describe("footer", () => {
    it("renders", () => {
        const { asFragment } = render(<Footer />);

        expect(asFragment()).toMatchSnapshot();
    });
});
