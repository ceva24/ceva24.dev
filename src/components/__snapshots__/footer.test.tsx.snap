// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="min-w-max text-center"
    >
      <a
        aria-current="page"
        class=""
        href="/"
      >
        Home
      </a>
    </div>
    <div
      class="my-8 text-center space-y-3 pt-5 border-t max-w-lg mx-auto"
    >
      <div
        class="font-normal"
      >
        CHRIS EVANS
      </div>
      <div
        class="px-6"
      >
        A Web Development / Systems Integration Team Leader at the University of York, UK
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
